packages/dnb-eufemia/src/components/accordion/AccordionDocs.ts

```diff
@@ -0,0 +1,155 @@
+import { PropertiesTableProps } from '../../shared/types'
+
+export const accordionProperties: PropertiesTableProps = {
```

Comment:
```suggestion
export const AccordionProperties: PropertiesTableProps = {
```

For now, most of the typed docs run with a PascalCase naming, so its the same as the "Component" name.

`<PropertiesTable props={AccordionProviderGroupProperties} />` etc. needs to be adjusted too.